rele/subscription.py

```diff
@@ -145,7 +158,14 @@ def __call__(self, message):
             run_middleware_hook("post_process_message")
 
 
-def sub(topic, prefix=None, suffix=None, filter_by=None, backend_filter_by=None):
+def sub(
+    topic,
+    prefix=None,
+    suffix=None,
+    filter_by=None,
+    backend_filter_by=None,
+    deserialize=lambda x: json.loads(x.data.decode("utf-8")),
```

Comment:
```suggestion
    deserialize=lambda x: json.loads(x.data.decode("utf-8")),
```

Im thinking about the api for this. Contrary to what I recommended before, I suggest this to be `deserializer` instead. As its a thing and not a verb. WDYT?

Reply:
Agreed. deserialize sounds like a boolean